src/components/NotificationDetail.tsx

```diff
@@ -0,0 +1,58 @@
+import Link from "next/link";
+
+type NotificationProps = {
+  key: number;
+  read: boolean;
+  kind: string;
```

Comment:
kind って3種類しかないという理解でOKですか？
それなら、ユニオン型を使ってみると良いかもしれません。

```suggestion
  kind: "application" | "approval_pending" | "unapplied";
```

Reply:
https://github.com/tatata05/next-app/pull/10/commits/bbe93498b614cc3980f4a933b68cf984ab7b223f
ユニオン型を使ってみました